hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: patch
+
+This patch stops Hypothesis from leaking implementation details - each
```

Comment:
I'm not sure "letting tracebacks behave like they normally do" counts as leaking information details.